packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -23,7 +24,10 @@ class ScrollBehavior {
   /// The platform whose scroll physics should be implemented.
   ///
   /// Defaults to the current platform.
-  TargetPlatform getPlatform(BuildContext context) => defaultTargetPlatform;
+  TargetPlatform getPlatform(BuildContext context) {
+    return MediaQuery.of(context, nullOk: true)?.platform
```

Comment:
Gotcha. I was thinking of changing the method back to a 1 liner using =>
Switched to the 83 char version.